jenkins/values.yaml

```diff
@@ -1,5 +1,6 @@
 ---
 appName: &name jenkins
+appHostName: jenkins.apps.hivec.sandbox509.opentlc.com
 
```

Comment:
probs should try get the hostname from the current cluster or not default it to your open tlc box

Reply:
you awake ? midnight .. lolz.. er yeah .. do you know how to do that with helm/argo ?

Reply:
OKAY, this works .. need to the right annotation for OCP and set host: to null